src/main/osd/osd.c

```diff
@@ -504,8 +504,7 @@ static int32_t getAverageEscRpm(void)
         for (int i = 0; i < getMotorCount(); i++) {
```

Comment:
This code us duplicated in `getMotorAveragePeriod` for slrx (but with inverted motor/esc priority.

Reply:
Fixed